openshift/s3-backup/docker/README.md

```diff
@@ -42,3 +42,15 @@ brew reinstall zlib bzip2
 poetry env use /Users/[user]/.pyenv/versions/3.6.8/bin/python
 poetry install
 ```
+
+### Ubunut specific instructions
+
+If you have trouble getting poetry to use python 3.6.*
```

Comment:
I thought we'd upgraded to a new version of python? Like 3.8 or something?

Reply:
For our API and Ubuntu - yes. For the backup image - no.